tools/mnist-playground/mnist-playground.cpp

```diff
@@ -0,0 +1,299 @@
+//===- mnist-playground.cpp -------------------------------------*- C++ -*-===//
+//
+// This file is licensed under a pytorch-style license
```

Comment:
I think I led you astray. What we are doing currently is if it is under the frontends/pytorch directory, we use this license block. Otherwise, use the normal LLVM license block. I need to send a message soon so we can decide what we are "really" doing with this (while contributors are small and we can change it).